src/core.js

```diff
@@ -44,8 +44,10 @@ function createModule( name, testEnvironment ) {
 		name: moduleName,
 		parentModule: parentModule,
 		tests: [],
+		unskippedTests: [],
 		moduleId: generateHash( moduleName ),
 		testsRun: 0,
+		unskippedTestsRun: 0,
```

Comment:
`testsRun` isn't used for anything other than determining whether to invoke module hooks, so we can just use that instead of introducing a new variable. Basically, we should count a test that actually ran should be an unskipped test. Make sense?